packages/mermaid/src/utils.spec.ts

```diff
@@ -594,3 +596,23 @@ describe('calculatePoint', () => {
     );
   });
 });
+
+describe('getSubGraphTitleMargins', () => {
+  it('should get subgraph title margins after config has been set', () => {
+    const config_0 = {
+      flowchart: {
+        subGraphTitleMargin: {
+          top: 10,
+          bottom: 5,
+        },
+      },
+    };
+
+    configApi.setSiteConfig(config_0);
+    expect(getSubGraphTitleMargins()).toEqual({
+      subGraphTitleTopMargin: 10,
+      subGraphTitleBottomMargin: 5,
+      subGraphTitleTotalMargin: 15,
+    });
+  });
+});
```

Comment:
Would prefer this in `utils/subGraphTitleMargins.spec.ts` instead.
`utils.spec.ts` is overcrowded now, and there's a plan to split it up. 

Also, please rename the file `getSubGraphTitleMargins.ts` => `subGraphTitleMargins.ts`

Reply:
Done.